obal/data/playbooks/release/release.yaml

```diff
@@ -4,6 +4,10 @@
   serial: 1
   gather_facts: no
   roles:
-    - diff_package
+    - role: diff_package
+      diff_package_tags: "{{ build_package_tags }}"
```

Comment:
where is `build_package_tags` coming from?

Reply:
That's old, will update